parsing/dialects/dialect1/config.fmpp

```diff
@@ -1010,6 +1015,7 @@ data: {
     # Example: SqlParseTimeStampZ().
     dataTypeParserMethods: [
       "ByteIntType()"
+      "BlobDataType()"
```

Comment:
nit: Let's put these in alphabetical order unless we feel there's a strong reason to try one before the other during parsing.